### ISSUE TITLE
only http-client.env.json gets recognized, instead of all *.env.json files. 

### ISSUE THREAD

**Issue by jg-cantaa** [AnWeber/vscode-httpyac#94]
![Screenshot_2021-12-08_14-32-29](https://user-images.githubusercontent.com/95756912/145218265-ceba533a-c245-4503-90bf-af4d9f1362d6.png)
As you can see, the test4/5/6 environments do not get recognized.

However, i don't know how IntelliJ handles multiple files declaring the same environment. Maybe disambiguate by calling the environments something like "rest-client: dev" for a "dev" environment defined in "rest-client.env.json".

**Comment by AnWeber** [comment]
Intellij mentions only http-Client.env.json file in their docs: https://www.jetbrains.com/help/webstorm/exploring-http-syntax.html#example-working-with-environment-files. I need to know, in which order they are interpreted, if same variable is defined to implement it.

**Comment by AnWeber** [comment]
Intellij finds really arbitrary `*.env.json` files. Unfortunately this was not documented. Intellij does not do a merge, but rejects duplicate environments with an error message.
![image](https://user-images.githubusercontent.com/13434739/145264762-0fd030e7-fb6c-4846-9de3-c45a2fccdef1.png)

**Comment by AnWeber** [comment]
Version 4.8.0 recognizes all *.env.json files as Intellij environment files. Thanks for the hint to this bug:-)

**Comment by jg-cantaa** [comment]
Thanks alot for the quick turnaround! 
But sadly, now it doesn't find any *.env.json files:
![Screenshot_2021-12-10_11-21-45](https://user-images.githubusercontent.com/95756912/145558584-019d9e5a-badb-4ee8-84e9-7e043beba34b.png)
(.env.test is empty)

**Comment by AnWeber** [comment]
Can you just zip your example. In my example it does work. https://github.com/httpyac/httpyac.github.io/tree/main/examples/project/intellijEnvironments.

**Comment by jg-cantaa** [comment]
Here you go:
[#94.zip](https://github.com/AnWeber/vscode-httpyac/files/7692016/94.zip)

**Comment by AnWeber** [comment]
Do you have special settings in the user settings?
![image](https://user-images.githubusercontent.com/13434739/145559860-d3c0c266-1938-457e-8507-16f2a87fbd62.png)

**Comment by AnWeber** [comment]
there is the error:-)
![image](https://user-images.githubusercontent.com/13434739/145560332-70dacf17-98bf-46d7-9fb7-49ba12aaeac4.png)

**Comment by jg-cantaa** [comment]
The Workspace settings are empty and the global settings json does not contain any httpyac settings

**Comment by AnWeber** [comment]
This also explains your other bug with envDirName. And it is relatively complicated. But the reason for this is mainly the CLI tool. This can be run on any level and should still find the env files. But then I got issues because it behaves differently. Therefore I use now the logic also here in VSCode.

**Comment by jg-cantaa** [comment]
But even with 
```
{
  "httpyac.environmentPickMany": false,
  "httpyac.useMethodInSendCodeLens": true
}
``` 
in the settings.json it does not find it :/

**Comment by AnWeber** [comment]
I don't understand that now. I have now exactly, your project used and reopened and then created the screenshot. Reopen the file/ VSCode and test again?

**Comment by jg-cantaa** [comment]
Nope, still not. Even after VSC restart.

Will try now with all but the httpyac extension.

**Comment by jg-cantaa** [comment]
Even with only httpyac extension enabled, it still doesn't work. 

Version: 1.62.3
Commit: ccbaa2d27e38e5afa3e5c21c1c7bef4657064247
Date: 2021-11-17T08:00:36.721Z
Electron: 13.5.2
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Linux x64 5.13.0-7620-generic

**Comment by AnWeber** [comment]
Not VSCode or httpyac. I am on Windows and you are on Linux, but this should be also cause no trouble. But will test it in WLS.

**Comment by jg-cantaa** [comment]
I have tried it now on my windows machine as well, still no luck.

**Comment by AnWeber** [comment]
No, I have found the error. I assumed that I get the absolute path from the `readdir` method, however this is only the filename. Because I have the http file in the same folder it worked. But why I was able to generate the screenshot earlier is still not clear to me, but now I also get the error.

**Comment by AnWeber** [comment]
Actually, I would have fixed it and wanted to provide you with an update right away, but somehow it hangs on NPM. There does not appear the version, although it was [published successfully](https://github.com/AnWeber/httpyac/runs/4483024888?check_suite_focus=true). I'll add it as soon as it's working again.

**Comment by AnWeber** [comment]
The bug has been fixed with the latest update (v4.8.1)

**Comment by jg-cantaa** [comment]
Thanks for the quick fix!
